src/main/java/com/salessparrow/api/domain/SalesforceOauthToken.java

```diff
@@ -18,7 +19,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @DynamoDBTable(tableName = "salesforce_oauth_tokens")
-public class SalesforceOauthToken {
+public class SalesforceOauthToken implements Serializable {
```

Comment:
Lets add a comment on why do we need Serializable

Reply:
Added comment